app/views/HouseConsignmentView.scala.html

```diff
@@ -39,6 +40,12 @@
 
     <p class="govuk-body">@messages("houseConsignment.paragraph")</p>
 
+    @govukSummaryList(
+        SummaryList(
+            rows = Seq(houseConsignmentViewModel.safetyAndSecurityRow)
+        )
+    )
```

Comment:
Could we not have restructured the House consignment view model to have the existing sections as `children` and the row you have added in the `rows`?